modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service-power-z.adoc

```diff
@@ -0,0 +1,112 @@
+[id="sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service-ibm-power-z_{context}"]
+= Connecting the Spring PetClinic sample application to the PostgreSQL database service
+
+To connect the sample application to the database service, you must create a `ServiceBinding` custom resource (CR) that triggers the {servicebinding-title} to project the binding data into the application.
+
+[discrete]
+.Procedure
+
+. Create a `ServiceBinding` CR to project the binding data:
++
+[source,terminal]
+----
+$ oc apply -f - << EOD
+---
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services: <1>
+    - group: postgresql.dev4devs.com
+      version: v1beta1
+      kind: Database <2>
+      name: sampledatabase
+      version: v1alpha1
+    application: <3>
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+EOD
+----
+<1> Specifies a list of service resources.
+<2> The CR of the database.
+<3> The sample application that points to a Deployment or any other similar resource with an embedded PodSpec.
++
+The output verifies that the `ServiceBinding` CR is created to project the binding data into the sample application.
++
+[source,terminal]
```

Comment:
Line 40 and Line 41 should be flipped around like so:

.Example output 
[source,terminal]